README.md

```diff
@@ -1,65 +1,93 @@
 
+Open a terminal and run:
 ```bash
-pip install streamlit
-streamlit hello
+$ pip install streamlit 
+$ streamlit hello
 ```
 
-Streamlit can also be installed in a virtual environment on [Windows](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-windows), [Mac](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux), and [Linux](https://github.com/streamlit/streamlit/wiki/Installing-in-a-virtual-environment#on-mac--linux).
+This should open our sweet _Streamlit Hello_ app right in your browser. It features a bunch of examples of what you can do with Streamlit!
 
-## A little example
+<img src="https://media.cleanshot.cloud/media/55064/9RAuEZKubvrlrlPVua50xuhyPIiD8MBdvzi7Pa3s.gif?Expires=1674790062&Signature=fqIRLW0tMXntC1vCrSq3yLPTtA58SRLiO1i0y~7YshYuOfQQokc0OI6uxWg6IBL8rw2gGnzA4ML-OdGWPgK6WSulyLR-wNbQAPiKX4LF0vXS9F6AmZ8V39Tze5Fo9TWfA3JZF3mnu7q6JrkssCk~6614xuugx9gojkOJMtsbFrOiM2kNLkje2OepBbxromYKVLLEQGwuPcFvUiA3Hb6ZuSHtNX~99wLu94PAKpXfPUf21sCt4vfmYItkl0z6BuHWYP5Zs5SlvPo8iXvdrb-dKL-~-kbTRjpcTFgWhFT1zg8iNZNCeR0Q~QvpS8rzfQPMMeIiBiRJFmTm4RC7~FRktA__&Key-Pair-Id=K269JMAT9ZF4GZ" alt="Streamlit Hello" width=500 href="none"></img>
 
-Streamlit makes it incredibly easy to build interactive apps:
+Read more on getting started with Streamlit in our [documentation](https://docs.streamlit.io/library/get-started).
 
-```python
-import streamlit as st
 
-x = st.slider('Select a value')
-st.write(x, 'squared is', x * x)
-```
+## Quick start
 
-<img src="https://raw.githubusercontent.com/streamlit/docs/main/public/images/simple_example.png"/>
+#### A little example
 
-## A bigger example
-
-Streamlit's simple and focused API lets you build incredibly rich and powerful tools.  [This demo project](https://github.com/streamlit/demo-self-driving) lets you browse the entire [Udacity self-driving-car dataset](https://github.com/udacity/self-driving-car) and run inference in real-time using the [YOLO object detection net](https://pjreddie.com/darknet/yolo).
-
-![Final App Animation](https://raw.githubusercontent.com/streamlit/docs/main/public/images/complex_app_example.gif)
-
-The complete demo is implemented in less than 300 lines of Python. In fact, the app contains [only 23 Streamlit calls](https://github.com/streamlit/demo-self-driving/blob/master/streamlit_app.py) which illustrates all the major building blocks of Streamlit. You can try it right now at [share.streamlit.io/streamlit/demo-self-driving](https://share.streamlit.io/streamlit/demo-self-driving).
+Streamlit makes it incredibly easy to build interactive apps:
 
-## The Streamlit GitHub badge
+```python
+import streamlit as st
+x = st.slider("Select a value")
+st.write(x, "squared is", x * x)
+```
 
-Streamlit's GitHub badge helps others find and play with your Streamlit app.
+<img src="https://user-images.githubusercontent.com/7164864/214900507-1c89fc6b-e196-4f5c-890f-7e0be62d5d9d.png" width=300 alt="Little example"></img>
```

Comment:
Same comment on the browser window as above. Also, I think it would be super cool to animate this demo. I.e. just a little gif that drags the slider and shows the app rerunning.

Reply:
Changed to a GIF but no solution for the actual window